package.json

```diff
@@ -134,7 +134,7 @@
     "express": "^4.15.4",
     "fun-hooks": "^0.9.9",
     "just-clone": "^1.0.2",
-    "live-connect-js": "^6.3.2"
+    "live-connect-js": "^6.3.4"
```

Comment:
Unrelated, but we should talk about it soon: `^` means, when you install the dependencies, the build system will automatically check for a newer version 6.x.y. That mean when we release e.g. a new patch of LiveConnect, but do not want it to be rolled out yet, but want to test it with a limited number of users, Prebid might just run their build and the newer version will be pulled in - something that we did not intend.